src/cmd/script.rs

```diff
@@ -126,13 +126,48 @@ pub fn new<'a, 'b>() -> App<'a, 'b> {
                     "common",
                 ])
         )
+        .arg(
+            Arg::with_name("package")
+                .short("p")
+                .long("package")
+                .help("Specify package to show sources for.")
```

Comment:
```suggestion
                .help("Specify package to show sources for")
```
No other help string for `script` ends with a period (absolute nitpicking, just struck my eye when I ran `script -h` :slightly_smiling_face:).